velox/exec/ContainerRowSerde.h

```diff
@@ -44,6 +44,15 @@ class ContainerRowSerde {
       const Type* type,
       CompareFlags flags);
 
+  /// Return std::nullopt if encountered in NullHandlingMode::StopAtNull mode.
```

Comment:
I find this comment a bit cryptic. Maybe, use the comment from BaseVector::compare as a base.

```
 /// Returns < 0 if 'left' is less than 'right' at 'index', 0 if
 /// equal and > 0 otherwise. If flags.nullHandlingMode is StopAtNull, 
 /// returns std::nullopt if either 'left' or 'right' value is null or contains a null.
```

Also, let's document existing 'compare' methods to clarify that these support only NoStop for flags.nullHandlingMode, consider NULL equal to NULL.

Reply:
Added